recipe/meta.yaml

```diff
@@ -42,33 +44,31 @@ requirements:
     - setuptools
     - wheel
   run:
-    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('cxx') }}
     - clang                  # [win]
     - clangxx                # [win]
     - python
     - numpy
     - gast >=0.5.0,<0.6.0
     - ply >=3.4
     - beniget >=0.4.0,<0.5.0
-    - setuptools # used in pythran.dist
-    - xsimd 12.1.1
+    - setuptools <74 # used in pythran.dist. distutils.msvccompiler removed in recent setuptools
 
 test:
   requires:
     - pip
     # default to testing openblas
     - openblas-devel {{ openblas }}   # [linux or osx]
-    - {{ compiler('c') }}  # [osx]
   files:
     - dprod.py
     - simple_numexpr.py
   commands:
     - pip check
-    - export CBS="-isysroot $(xcrun --show-sdk-path)" # [osx]
-    - export CFLAGS="${CFLAGS} ${CBS}"  # [osx]
-    - export CXXFLAGS="${CXXFLAGS} ${CBS}"  # [osx]
-    - export LDFLAGS="${LDFLAGS} ${CBS}"  # [osx]
     - set "SRC_DIR="  # [win]
+    # Set compilers to clang-cl
+    - set "CC=clang-cl.exe"  # [win]
+    - set "CXX=clang-cl.exe" # [win]
+    - set "LDFLAGS=" # [win]
```

Comment:
so do these all need to be set whenever it runs? Do we need an activation script here? I assume we're handling it in scipy and scikit-image, but any users that want to use pythran directly will probably run into issues

Reply:
The activation script should be in `clang_win-64`. But there are none at the moment.